spec/datadog/tracing/contrib/propagation/sql_comment_spec.rb

```diff
@@ -102,6 +103,63 @@
       end
     end
 
+    context 'when tracing is enabled with peer.service' do
```

Comment:
The description does not match the scenario, it only need a span_op  with a specific tag

Reply:
I disagree, the main thing here is to mock a situation when tracing is enabled with `peer.service`, ensuring that the `peer.service` value is getting set as the `dddbs` tag.

> it only need a span_op with a specific tag

That's what the peer.service feature does. It adds a tag

Reply:
Sorry, I meant that the test could be simplified, for example, 

We don't need 
```
        Datadog.configure do |c|
          c.env = 'production'
          c.service = "Traders' Joe"
          c.version = '1.0.0'
        end
```

Reply:
The description should be 
```
context 'when given a span operation tagged with`peer.service`' do
```